src/index.css

```diff
@@ -7,6 +7,10 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+::-webkit-scrollbar {
```

Comment:
When the page gets longer, scrollbar shows and whole pages moves a bit. For now I've hidden this scrollbar, but we can decide to 1) leave moving page as it is 2) force the scrollbar to show all the time.

Reply:
It's a non-urgent thing, we can handle it later. I would create a different ticket.

Reply:
Created the ticket.